Controller/TreeController.php

```diff
@@ -22,9 +21,9 @@
 class TreeController extends Controller
```

Comment:
do we still need such a controller in this bundle or will this all move to the tree bundle?